level-2/index.html

```diff
@@ -192,7 +192,8 @@ <h4>
             <li>
               <a data-cite="!HTML#in-parallel">In parallel</a>:
               <ol>
-                <li>If there are no <a>share targets</a> available,
+                <li>If there are no <a>share targets</a> available, or a file
+                type is being blocked due to security considerations,
```

Comment:
nit: Does this fit better with the step above which checks for user activation?

Reply:
Updated to reject with a NotAllowedError.